api/go.mod

```diff
@@ -70,3 +70,5 @@ require (
 // mschuppert: map to latest commit from release-4.16 tag
 // must consistent within modules and service operators
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20240830023148-b7d0481c9094 //allow-merging
+
+replace github.com/openstack-k8s-operators/lib-common/modules/common => github.com/fmount/lib-common/modules/common v0.0.0-20250121110137-f3ec1120ae76 //allow-merging
```

Comment:
just a side note, adding `allow-merging` to this kind of replace is dangerous as the pre-commit replace check won't report it and the PR could be merged by the automation

Reply:
I can remove it, sure. I thought the `do-not-merge` label was enough to prevent the automation merging this patch automatically. I'll remove it in my next iteration.

Reply:
yes sure, the do not merge is enough. its not critical, just wanted to mention that basically the `allow-merging` will override our pre-commit protection check.

Reply:
Removed